package.json

```diff
@@ -9,8 +9,8 @@
     "babel-loader": "^8.0.4",
     "babel-plugin-add-react-displayname": "0.0.5",
     "babel-preset-es2015": "^6.24.1",
-    "babel-preset-react": "^6.24.1",
     "babel-preset-react-app": "^6.0.0",
+    "babel-preset-react": "^6.24.1",
```

Comment:
the ordering feels like it was edited manually, Its better to do `npm install package@version` and let npm do the job of adding it to package.json.